src/test/java/uk/gov/ons/ctp/response/action/state/TestActionStateTransitionManager.java

```diff
@@ -92,6 +92,10 @@ public void setup() {
     final Map<ActionEvent, ActionState> cancelledTransitions = new HashMap<>();
     cancelledTransitions.put(ActionEvent.REQUEST_CANCELLED, ActionState.CANCELLED);
     validTransitions.put(ActionState.CANCELLED, cancelledTransitions);
+
+    final Map<ActionEvent, ActionState> reRunTransitions = new HashMap<>();
+    reRunTransitions.put(ActionEvent.REQUEST_RERUN, ActionState.SUBMITTED);
```

Comment:
It makes no sense to me that we have a test version of the transition manager. Surely the transitions which are valid in production are the same ones that are valid in our tests?

I know you didn't create these tests like this originally, but it's madness to not test the actual production code, but instead be testing a class which isn't the real one.

Reply:
Yeah as far as I'm aware I just went with what was "already put in place"
So defo would be worth splitting this out into a separate card that we could discuss and bring in?

Reply:
Well, at the moment you don't have a test to check that the state can be transitioned from ABORTED to SUBMITTED via REQUEST_RERUN... you're just testing this test code.